bindata/network/openshift-sdn/alert-rules.yaml

```diff
@@ -39,40 +39,39 @@ spec:
       for: 1h
       labels:
         severity: warning
-    - alert: IPTableSyncSDNPod
+    - alert: NodeProxyApplySlow
       annotations:
-        message: SDN pod {{"{{"}} $labels.pod {{"}}"}} on node {{"{{"}} $labels.node {{"}}"}} takes too long to sync iptables rules.
+        message: SDN pod {{"{{"}} $labels.pod {{"}}"}} on node {{"{{"}} $labels.node {{"}}"}} is taking too long to apply iptables rules.
       expr: |
-        histogram_quantile(.95, kubeproxy_sync_proxy_rules_duration_seconds_bucket) * on(pod) group_right kube_pod_info{namespace="openshift-sdn",  pod=~"sdn-[^-]*"} > 15
+        histogram_quantile(.95, kubeproxy_sync_proxy_rules_duration_seconds_bucket) 
+        * on(pod) group_right kube_pod_info{namespace="openshift-sdn",  pod=~"sdn-[^-]*"} > 15
       labels:
         severity: warning
-    - alert: IPTableSyncCluster
+    - alert: ClusterProxyApplySlow
       annotations:
-        message: The average time for SDN pods to sync iptables is too high.
+        message: The cluster is taking too long, on average, to apply iptables rules.
       expr: |
         histogram_quantile(0.95, sum(rate(kubeproxy_sync_proxy_rules_duration_seconds_bucket[5m])) by (le)) > 10
       labels:
         severity: warning
-    - alert: NodeIPTablesStale
-    # there is some scrape delay and some other offset 120 is not really 120s but it is still too long
+    - alert: NodeProxyApplyStale
       annotations:
         message: SDN pod {{"{{"}} $labels.pod {{"}}"}} on node {{"{{"}} $labels.node {{"}}"}} has gone too long without syncing iptables rules.
```

Comment:
Perhaps this is is a bit clearer to end users?
         message: SDN pod {{"{{"}} $labels.pod {{"}}"}} on node {{"{{"}} $labels.node {{"}}"}} is stale syncing iptables service rules.